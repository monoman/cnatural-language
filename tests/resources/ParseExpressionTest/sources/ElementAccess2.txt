t[1, u[2]]