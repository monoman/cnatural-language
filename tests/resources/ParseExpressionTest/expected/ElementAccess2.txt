(Name(t))[Literal(DecimalInteger, [1]), (Name(u))[Literal(DecimalInteger, [2])]]